brain/validator/validate_password.py

```diff
@@ -0,0 +1,130 @@
+'''@validate_password
+
+This file contains functions for validating the user's password against
+a set of requirements.
+
+'''
+
+from flask import current_app
+import yaml
+
+
+def load_min(app=True, root='/vagrant'):
+    '''@load_min
+
+    This method returns the minimum character requirement for passwords.
+
+    @app, indicates if function is to be used by application, or manually
+
+    '''
+    if app:
+        min_c = current_app.config.get('PASSWORD_MIN_C')

```

Comment:
I would change `current_app.config.get('PASSWORD_MIN_C')` to `current_app.config.get('PASSWORD_MIN_C', None)`.  This way, instead of logic raising an `KeyError`, it will default to `None` if no keys was found via `get`.